core/src/main/kotlin/endpoints/AgentManager.kt

```diff
@@ -545,54 +537,54 @@ class AgentManager(override val kodein: Kodein) : KodeinAware {
         notifySingleAgent(agentId)
     }
 
-    private suspend fun wrapInstanceBusy(
-        instanceKey: InstanceKey,
+    private suspend fun AgentInfo.wrapInstanceBusy(
+        instanceId: String,
         block: suspend AgentWsSession.() -> Unit,
-    ): Unit = getInstanceState(instanceKey)?.let { instanceState ->
-        val agentId = instanceKey.agentKey.agentId
-        updateInstanceStatus(instanceKey, AgentStatus.BUSY).also { instance ->
-            if (instance?.all { it.value.status == AgentStatus.BUSY } == true) {
-                notifyAgents(agentId)
-                logger.debug { "Agent $agentId is busy." }
+    ): Unit = getInstanceState(instanceId)?.let { instanceState ->
+        updateInstanceStatus(instanceId, AgentStatus.BUSY)?.also { instance ->
+            if (instance.all { it.value.status == AgentStatus.BUSY }) {
+                notifyAgents(id)
+                logger.debug { "Agent $id is busy." }
```

Comment:
should we add buildVersion to logging for easy investigation of a problem?

Reply:
I would like to add instance id to logs, for example to this:
 `DEBUG c.e.d.admin.endpoints.AgentManager - Sending 1 plugins to agent Petclinic
DEBUG c.e.d.admin.endpoints.AgentManager - Sending plugin test2code to agent Petclinic
DEBUG c.e.d.admin.endpoints.AgentManager - Sent data of plugin test2code to agent Petclinic
DEBUG c.e.d.admin.endpoints.AgentManager - Sent plugin test2code to agent Petclinic
DEBUG c.e.d.admin.endpoints.AgentManager - Sent plugins [test2code] to agent Petclinic
DEBUG c.e.d.admin.endpoints.AgentManager - Enabling all plugins for agent with id Petclinic
DEBUG c.e.d.admin.endpoints.AgentManager - Enabling plugin test2code for agent Petclinic...`  
adding buildVersion makes sense too

Reply:
There are more changes than I thought. Maybe expand the logs into another task?